packaging/ubuntu-16.04/changelog

```diff
@@ -1,3 +1,10 @@
+snapd (2.59.5fix1) UNRELEASED; urgency=medium
+
+  * snapd-generator vs mountinfo fix
+  * snapd-generator vs mountinfo in systemd sandbox fix
+
+ -- Ernest Lotter (Canonical Services Ltd.) <ernest.lotter@canonical.com> Thu, 02 Nov 2023 10:38:41 +0200
```

Comment:
I'm not entirely sure if this bit is useful or not, as if we do a new release will need a 2.59.6 anyway